idlc/src/main.rs

```diff
@@ -155,6 +159,11 @@ fn main() {
     trace!("Verifying interfaces");
     interface_verifier::InterfaceVerifier::new(&mir).run_pass();
 
+    let marking = idlc_codegen::documentation::Documentation::add_marking(
```

Comment:
We can't always add `C` markings this needs to be specific to the language we need to emit